pluma/pluma-view.c

```diff
@@ -181,6 +183,36 @@ document_read_only_notify_handler (PlumaDocument *document,
 				    !pluma_document_get_readonly (document));
 }
 
+static gboolean
+pluma_view_scroll_event (GtkWidget      *widget,
+			 GdkEventScroll *event)
```

Comment:
indentation?

Reply:
It looks fine here: https://raw.githubusercontent.com/mate-desktop/pluma/1cd757ab5b2688f9628bd97d1a634020b544379e/pluma/pluma-view.c

The tabs... A lot of functions have the same indentation.

I will fix it later with only spaces.